internal/store/persistentvolume.go

```diff
@@ -153,6 +156,7 @@ var (
 								"iscsi_target_portal",
 								"iscsi_iqn",
 								"iscsi_lun",
+								"iscsi_initiator_name",
```

Comment:
Is this just a weird way github is displaying a diff or is it a formatting issue?

Reply:
i think it might just be the display.  if you view the file (vs. the diff), does it look as you would expect?

Reply:
> Is this just a weird way github is displaying a diff or is it a formatting issue?

i think it might just be the display. if you view the file (vs. the diff), does it look as you would expect?